packages/okta-react/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.0.4
```

Comment:
I (again) thought we had decided on a major version bump (see comments from Aaron) - what am I missing?

Reply:
I think we decide to make remove default `onSessionExpired` as a patch fix to avoid noise. Then put the `isAuthenticated` behavior change into 4.0.

Reply:
Right. We agreed that removing `onSessionExpired` is a **fix** not a new feature or breaking change. It was never working as intended, and has mostly negative side effects. The positive side effect was triggering a login if **only** the accessToken is missing, but this can be done better by the user providing an `isAuthenticated` function which asserts `accessToken && idToken`. We will change this to be default logic in the next version but it would likely be considered a breaking change.

Reply:
> See comments.
> 
> * major version bump
> * remove onSessionExpired
> * change logic on isAuthenticated to return false if either token is missing
> * double check for concurrency errors with useEffect -> login()

As we discussed we will push breaking change with `isAuthenticated` to subsequent release that includes `okta-auth-js@4.0`.

I think we can update the README to provide better guidance for users of this version. (see comments).